packages/taquito-rpc/src/types.ts

```diff
@@ -1732,7 +1732,8 @@ export interface ConstantsResponseProto020
     | 'preserved_cycles'
     | 'smart_rollup_reveal_activation_level'
   > {
-  adaptive_issuenace_force_activation: boolean;
+  adaptive_issuance_activation_vote_enable: boolean;
```

Comment:
sorry for misleading you `adaptive_issuance_activation_vote_enable` is included in proto019